index.html

```diff
@@ -3330,9 +3330,9 @@ <h2>Understanding Proof Sets and Proof Chains</h2>
 <em>secretVPE/publicVPE</em>, respectively.
       </p>
       <p>
-When multiple signatories intend to sign an |inputDocument| and the ordering of
-those signatures are not of concern and there are no dependencies amongst the
-signatures then a [=proof set=] is typically a more straight forward
+When multiple signatories intend to sign an |inputDocument|, the ordering of
```

Comment:
I should have done this before. If/then construction is more common than when/then.
```suggestion
If multiple signatories intend to sign an |inputDocument|, the ordering of
```